ed448/src/sign/signing_key.rs

```diff
@@ -324,40 +325,48 @@ impl pkcs8::EncodePrivateKey for KeypairBytes {
         private_key[1] = SECRET_KEY_LENGTH as u8;
         private_key[2..].copy_from_slice(self.secret_key.as_ref());
 
-        let private_key_info = pkcs8::PrivateKeyInfo {
-            algorithm: super::ALGORITHM_ID,
-            private_key: &private_key,
-            public_key: self.verifying_key.as_ref().map(|v| v.as_ref()),
-        };
-        let result = pkcs8::SecretDocument::encode_msg(&private_key_info)?;
+        todo!();
+
+        // let private_key_info = pkcs8::PrivateKeyInfo {
+        //     algorithm: super::ALGORITHM_ID,
+        //     private_key: &private_key,
+        //     public_key: self.verifying_key.as_ref().map(|v| v.as_ref()),
+        // };
+
+        // No idea how to make these traits happy...
+        // let result = pkcs8::SecretDocument::encode_msg(&private_key_info)?;
```

Comment:
@tarcieri I have no idea how to update this for the latest pkcs8, the traits got a bit out of hand in the latest version I have a feeling